mushroomsoft/app/src/app/components/header/header.component.scss

```diff
@@ -251,7 +217,97 @@
     font-weight: 400;
 }
 
+@media screen and (max-width: 767px) {
+    .ms-nav[data-collapse="small"] .ms-nav-menu {
+        display: none
+    }
+
+    .ms-nav[data-collapse="small"] .ms-nav-button {
+        display: block
+    }
+
+    .ms-nav-brand {
+        padding-left: 10px
+    }
+}
+
+@media screen and (max-width: 479px) {
+    .ms-nav[data-collapse="tiny"] .ms-nav-menu {
+        display: none
+    }
+
+    .ms-nav[data-collapse="tiny"] .ms-nav-button {
+        display: block
+    }
+}
+
+@media screen and (max-width: 1024px) {
+    .ms-nav[data-collapse="medium"] .ms-nav-menu {
+        display: none
+    }
+
+    .ms-nav[data-collapse="medium"] .ms-nav-button {
+        display: block
+    }
+
+    .ms-button {
+        justify-content: center;
+    }
+
+    .ms-menu-icon {
+        display: block;
+    }
+
+    .ms-navigation-item {
+        padding: 15px 30px;
+        transition: background-color 400ms ease, opacity 400ms ease, color 400ms ease;
+        text-align: center;
+    }
+
+    .ms-menu-input{
+        display: none;
+    }
+
+    .ms-nav-link {
+        text-decoration: none;
+        width: 100%;
+    }
+
+    .ms-button{
+        margin: 0;
+        padding-left: 460px;
+
+    }
+
+    /* hidden menu */
+    .ms-nav-menu-phone{
+        position: fixed;
+        top:80px;
+        bottom: 0;
+        background: #ffffff;
+        width: 100%;
+        height: 21.2rem !important; 
```

Comment:
Try to avoid using "`!important`" in all occurrences.

Reply:
resolved